scripts/certgen.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# -----
+# Purpose: This script is used to Generate BIE kafka Compatible Base64 Encoded
+#   certificate string. This script also generates two passwords truststore and keystore passwords
+#
+# Output: Output of this script is output.json. 
+#   Output.json Contains 4 key value pairs which are then uploaded to vault
+#
+#   Following keys/values are generated
+#       BIE_KAFKA_KEYSTORE_INBASE64
+#       BIE_KAFKA_KEYSTORE_PASSWORD
+#       BIE_KAFKA_TRUSTSTORE_INBASE64
+#       BIE_KAFKA_TRUSTSTORE_PASSWORD
+# Pre-Req:
+#     - Download Certs locally. Link is provided in wiki https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/VRO-Secrets
+#       VA-Internal-S2-ICA4.cer 
+#       VA-Internal-S2-ICA19.cer
+#       VA-Internal-S2-ICA11.cer 
+#       VA-Internal-S2-RCA2.cer
+# Input: 
+# $1 Input environment name, Example dev/qa/prod
+# $2 Input BIE provided Certificate file
+# $3 Input BIE provided key
+# $4 Input Kafka env prefix, Example: dev,ivv,pr,pp
+# 
+# - ----
+
+# Function:  generates a random strong password
+#            used for generating random password for keystore and truststore
+# output: Random password string
+# input: None
+generate_password() {
+    # Define characters for password generation
+    upper="ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    lower="abcdefghijklmnopqrstuvwxyz"
+    numbers="0123456789"
+    special="@#\$%^&*()_-+="
+
+    # Combine all characters into one set
+    all_chars="${upper}${lower}${numbers}${special}"
+
+    # Generate a random password of length 15 using openssl
+    openssl rand -base64 20 | tr -dc "$all_chars" | head -c 16
+}
+
+export KEYSTORE_PWD=$(generate_password)
+export TRUSTSTORE_PWD=$(generate_password)
+
+# Delete temporary files ( if exist ) to avoid issues when re-running script
+# keeping them will result in password mismatch error. Its safe to delete as its recreated
+rm -f bip.truststore.jks keystore.p12 bip.truststore.p12 passwd output.json
+ 
+echo "Key Store pwd : $KEYSTORE_PWD"
+echo "Trust Store pwd: $TRUSTSTORE_PWD"
+
```

Comment:
Why is this needed? They are in output.json.
```suggestion
echo "Key Store pwd : $KEYSTORE_PWD"
echo "Trust Store pwd: $TRUSTSTORE_PWD"

```